actions/send-slack-message/README.md

```diff
@@ -78,7 +60,6 @@ jobs:
                 }
               ]
             }
-
           update-ts: ${{ steps.slack.outputs.ts }}
```

Comment:
This belongs as "ts" field into the payload 🙂 
https://github.com/slackapi/slack-github-action?tab=readme-ov-file#updating-a-message